edpop_explorer/record.py

```diff
@@ -243,6 +243,11 @@ class BiographicalRecord(Record):
     places_of_activity: Optional[List[Field]] = None
     timespan: Optional[Field] = None
     activities: Optional[List[Field]] = None
+    gender: Optional[List[Field]] = None
```

Comment:
`gender` should not be a `List`, I would say

Reply:
Ok, I will change it. I made it a list because there are several genders listed with entries of a bookseller and their children.